angular-client/src/app/app.module.ts

```diff
@@ -196,7 +199,9 @@ import { BatteryInfoDisplay } from 'src/pages/charging-page/components/battery-i
     MatToolbarModule,
     MatButtonModule,
     MatInputModule,
-    ReactiveFormsModule
+    ReactiveFormsModule,
+    CommonModule,
+    BrowserAnimationsModule // required animations module
```

Comment:
shouldn't need these changes anymore, since you're using Peyton's messageService, pls undo them